config/am.py

```diff
@@ -53,6 +53,73 @@
 NUMERALS = union(byte.DIGIT,
                  utils.GEEZ_NUMERALS)
 
+# Amharic "over-differentiates" H graphemes, emphatic S graphemes, and glottal
+# stop graphemes, which were all inherited from Ge'ez. Surveys suggest that
+# Amharic speakers prefer one form over the others. These rules convert the
+# dispreferred series graphemes to the one preferred series, when available.
```

Comment:
Just one more nit, could you add a link to the paper in the comment?  Thanks!

Reply:
Done!